exporter/awsemfexporter/README.md

```diff
@@ -33,10 +33,12 @@ The following exporter configuration parameters are supported.
 | `resource_to_telemetry_conversion`           | "resource_to_telemetry_conversion" is the option for converting resource attributes to telemetry attributes. It has only one config onption- `enabled`. For metrics, if `enabled=true`, all the resource attributes will be converted to metric labels by default. See `Resource Attributes to Metric Labels` section below for examples.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | `enabled=false` | 
 | `output_destination`                         | "output_destination" is an option to specify the EMFExporter output. Currently, two options are available. "cloudwatch" or "stdout"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | `cloudwatch` | 
+| `detailed_metrics`           | "detailed_metrics" is the options for retaining detailed datapoint values in exported metrics (e.g instead of exporting a quantile as a statistical value, preserve the quantile's population)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | `false` | 
```

Comment:
I'm fine with removing it since I saw others document such as output_destination having the same pattern
```
| `output_destination`                         | "output_destination" is an option to specify the EMFExporter output. Currently, two options are available. "cloudwatch" or "stdout"  
```